### PR TITLE
Bokeh widgets editable

### PR DESCRIPTION
**Pull request by philippjfr**
Adds an editable option, which is False by default to improve the look of the widgets, but because that disables support for enabling text input on sliders, can be set to True if needed.

### CODE REVIEW

**Comment by philippjfr** [comment]
Ready to review/merge.

File: holoviews/plotting/bokeh/widgets.py [hunk 1]
```diff
@@ -27,14 +27,8 @@ class BokehServerWidgets(param.Parameterized):
...
-    basejs = param.String(default=None, precedence=-1, doc="""
-        Defines the local CSS file to be loaded for this widget.""")
-
-    extensionjs = param.String(default=None, precedence=-1, doc="""
-        Optional javascript extension file for a particular backend.""")
-
-    css = param.String(default=None, precedence=-1, doc="""
-        Defines the local CSS file to be loaded for this widget.""")
+    editable = param.Boolean(default=False, doc="""
+        Whether the slider text fields should be editable.""")
```
Comment (jlstevens) [inline holoviews/plotting/bokeh/widgets.py]: Should mention the  the reason it is disabled by default. It sounds useful, so there is a reason we want it off by default...

**Comment by jlstevens** [comment]
Fairly short PR and other than updating the docstring, it looks good.

**Comment by jlstevens** [comment]
Tests passing. Merging.

**Pull request merged**